### DISCUSSION TITLE
Welcome to formsnap Discussions!

### DISCUSSION THREAD

**Discussion by huntabyte** [svecosystem/formsnap#103] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ramij333** [comment]
### Subject: Using Multiple Forms with Snapform — Need Help with `Form.Field`

Hi,

I'm working on a Svelte project where I need to handle multiple forms within a single component (e.g., `loginForm`, `otpForm`, etc.). However, I ran into an issue when trying to use multiple forms on the same page, since the `<Form.Field>` component accepts only a single `form` prop. For example:

```svelte
<Form.Field {form} name="xyz" class="w-3/5 space-y-4">
```
In my case, I would like to pass different forms (loginForm, otpForm, etc.) to <Form.Field>, but this doesn't seem to be possible with the current structure. I can't pass multiple forms like this:

```svelte
<Form.Field {loginForm} name="xyz" class="w-3/5 space-y-4">
```

This causes issues with handling multiple forms on the same page. Could you please provide guidance on how I can use multiple forms in Snapform while still maintaining proper form handling and validation for each?

For context, I want to manage different forms (like login, OTP verification, etc.) separately and use the correct form state with their respective fields. Is there a recommended pattern or solution for this use case?

Thank you!
